core/src/main/python/wlsdeploy/aliases/model_constants.py

```diff
@@ -175,6 +175,7 @@
 MAIL_SESSION = 'MailSession'
 MAIL_SESSION_OVERRIDE = 'MailSessionOverride'
 MAIL_SESSION_PROPERTIES = 'Properties'
+MAX_DYNAMIC_CLUSTER_SIZE = 'MaximumDynamicServerCount'
```

Comment:
it's minor, but can you make the variable name reflect the value? cluster has too many similar attributes related to dynamic server count.